### DISCUSSION TITLE
Error starting game

### DISCUSSION THREAD

**Discussion by enrico002** [wiremod/wire#2874] [Q&A] [answered]
When starting a game this message appears:

[Wiremod] Warning: Use of variadic parameter with ExtPP is not recommended and deprecated. Instead use ...<name> (which passes a table) or the `args` variable (at line 80) @entities/gmod_wire_expression2/core/custom/primitive.lua

    1. parseParameters - lua/entities/gmod_wire_expression2/core/extpp.lua:83
        2. Pass2 - lua/entities/gmod_wire_expression2/core/extpp.lua:194
            3. unknown - lua/entities/gmod_wire_expression2/core/extloader.lua:91
                4. pcall - [C]:-1
                    5. e2_include_finalize - lua/entities/gmod_wire_expression2/core/extloader.lua:110
                        6. unknown - lua/entities/gmod_wire_expression2/core/extloader.lua:183
                            7. include - [C]:-1
                                8. unknown - lua/entities/gmod_wire_expression2/core/init.lua:284
                                    9. include - [C]:-1
                                        10. unknown - lua/entities/gmod_wire_expression2/shared.lua:24
                                            11. include - [C]:-1
                                                12. unknown - lua/entities/gmod_wire_expression2/init.lua:3

**Comment by Denneisk** [comment, marked as answer]
If you're an extension developer:
- Change any occurances of variadic functions `e2function name(...)` to `e2function name(...args)`, which directly passes a table instead of having to cast it as `{...}`, which should no longer be relied on.

If you're a user:
- Contact the extension developer to update the extension OR just ignore it.
